tests/plugins/pastefromword/generated/excel.js

```diff
@@ -0,0 +1,55 @@
+/* bender-tags: clipboard,pastefromword */
+/* jshint ignore:start */
+/* bender-ckeditor-plugins: pastefromword,ajax,basicstyles,bidi,font,link,toolbar,colorbutton,image */
+/* bender-ckeditor-plugins: list,liststyle,sourcearea,format,justify,table,tableresize,tabletools,indent,indentblock,div,dialog */
+/* jshint ignore:end */
+/* bender-include: _lib/q.js,_helpers/promisePasteEvent.js,_helpers/assertWordFilter.js,_helpers/createTestCase.js */
+/* bender-include: _helpers/createTestSuite.js,_helpers/pfwTools.js */
+/* global createTestSuite,pfwTools */
+
+( function() {
+	'use strict';
+
+	var config = pfwTools.defaultConfig;
+	config.colorButton_normalizeBackground = true;
+	// Firefox adds `border-color:windowtext currentcolor windowtext windowtext;`
+	// for `td` elements with `colspan` attribute.
+	config.disallowedContent = 'td{vertical-align,border-color}';
```

Comment:
The `td{vertical-align}` was disallowed in `pfwTools.defaultConfig` which is used so I just added the `border-color` here. But I see it is not needed so your commit makes perfect sense.